torch/csrc/jit/backends/nnapi/nnapi_backend_preprocess.cpp

```diff
@@ -0,0 +1,45 @@
+#include <pybind11/pybind11.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_preprocess.h>
+#include <torch/csrc/jit/python/pybind_utils.h>
+
+namespace py = pybind11;
+
+// Converts model to nnapi and serializes it for mobile
+// Returns a dictionary string with one entry
+// Key: "NnapiModule"
+// Value: a string of the nnapi module, saved for mobile
+c10::IValue preprocess(
+    const torch::jit::Module& mod,
+    const c10::Dict<c10::IValue, c10::IValue>& method_compile_spec,
```

Comment:
Put an example of method_compile_spec: {"forward": {"inputs": Tensor}} in comment

Reply:
I still don't think the current documentation nor example are clear.

Reply:
Could you be a bit more specific? I think that would be really helpful! Maybe I should add more explanations for what input is for? Or clarify that compile_spec is a dictionary within a dictionary?

Reply:
Yes, basically the comments I made above.
Also, it would be good if the example shows how "Tensor" is created.